client/my-sites/stats/stats-subscribers-page/subsriber-highlights.jsx

```diff
@@ -0,0 +1,97 @@
+import { HighlightCard, Popover } from '@automattic/components';
+import { Icon, info } from '@wordpress/icons';
+import { translate } from 'i18n-calypso';
+import { useRef, useState } from 'react';
+import '../wordads/highlights-section.scss';
+
+function getSubscriberHighlights() {
+	const highlights = [
```

Comment:
7-day highlight from the Traffic page and Annual highlight from the Insights page are in Storybook. It's also a great place to use with mock data for testing. Let's move the component there using the same pattern from Traffic and Insights pages.

Reply:
Sounds good! I was looking at this again today to make sure I have the right level of separation. I think it makes sense to have all data configuration in the Stats code and only visual components in the Storybook to keep things from being coupled too tightly. I'll review the highlights we currently have in Storybook and address that in a follow-up PR.